.github/workflows/semver_checks.yml

```diff
@@ -56,8 +57,11 @@ jobs:
       id: semver-pr-check
       run: |
         set +e
-        make semver-rev rev="$PR_BASE"
-        exitcode=$?
+        echo "output<<SEMVER_STDOUT_EOF" >> $GITHUB_OUTPUT
+        # Weird sed strip ANSI colors from output
+        make semver-rev rev="$PR_BASE" |& tee /dev/tty | sed -r "s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g" >> $GITHUB_OUTPUT
```

Comment:
`CARGO_TERM_COLOR=never` should force the output to be emitted without colors, no need for complicated regexes.

Reply:
I wanted the colors to be visible in actions output (because it's prettier), but removed in report comment (because there is no way to render them)

Reply:
OK. I just wish there were a solution which doesn't require such incantations, but it's fine if it works.

Reply:
Yeah, I don't like it either, but I didn't find any easier way. If you know any, let me know